sql/src/main/java/io/crate/executor/transport/task/DropTableTask.java

```diff
@@ -109,7 +114,13 @@ public void onResponse(DeleteIndexResponse response) {
                 if (!response.isAcknowledged()) {
                     warnNotAcknowledged();
                 }
-                consumer.accept(RowsBatchIterator.newInstance(ROW_ONE), null);
+                userManager.dropTablePrivileges(tableIdent.fqn()).whenComplete((r, t) -> {
+                    if (t != null) {
+                        logger.warn("Unable to drop existing privileges for table {}. The privileges will  be " +
```

Comment:
hm I think we should at least log the exception. also the 2nd sentence is a bit confusing, I'd just leave it out.